packages/web/src/shared/components/form/ComboboxInput.tsx

```diff
@@ -6,97 +6,36 @@ import {
   useEffect,
   useState,
 } from 'react';
-import { Controller, useFormContext } from 'react-hook-form';
+import { useFormContext } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import { Icon } from '../Icon';
 
-const CREATE_TAG = '_create';
 const MAX_ITEMS = 1000;
 
 export interface ComboboxItem {
   label: string;
   value: string;
 }
 
-export type ComboboxInputProps = BaseComboboxInputProps & {
-  required?: boolean;
-  isolate?: boolean;
-};
-
-export default function ComboboxInput(props: ComboboxInputProps) {
-  const context = useFormContext();
-
-  if (context && !props.isolate) {
-    return (
-      <Controller
-        control={context.control}
-        name={props.name}
-        defaultValue={props.defaultValue}
-        rules={{ required: props.required }}
-        render={({ field, fieldState }) => (
-          <BaseComboboxInput
-            {...field}
-            items={props.items}
-            hasErrors={!!fieldState.error}
-          />
-        )}
-      />
-    );
-  } else {
-    const {
-      className,
-      name,
-      items,
-      value,
-      defaultValue,
-      hasErrors,
-      up,
-      onBlur,
-      onChange,
-      onCreate,
-      onKeyDown,
-    } = props;
-    return (
-      <BaseComboboxInput
-        className={className}
-        name={name}
-        items={items}
-        value={value}
-        defaultValue={defaultValue}
-        hasErrors={hasErrors}
-        up={up}
-        onBlur={onBlur}
-        onChange={onChange}
-        onCreate={onCreate}
-        onKeyDown={onKeyDown}
-      />
-    );
-  }
-}
-
 interface BaseComboboxInputProps
   extends Omit<ComponentProps<'input'>, 'value' | 'onChange' | 'ref'> {
   className?: string;
-  name: string;
+  name?: string;
   items: ComboboxItem[];
   value?: string;
   defaultValue?: string;
-  hasErrors?: boolean;
   up?: boolean;
   onBlur?(): void;
   onChange?(value: string): void;
-  onCreate?(text?: string): void;
```

Comment:
I also got rid of the onCreate event on the combobox since we aren't using it anymore